svgdigitizer/svgdigitizer.py

```diff
@@ -10,6 +10,7 @@
 ref_point_regex_str = r'^(?P<point>(x|y)\d)\: ?(?P<value>-?\d+\.?\d*) ?(?P<unit>.+)?'
 scale_bar_regex_str = r'^(?P<axis>x|y)(_scale_bar|sb)\: ?(?P<value>-?\d+\.?\d*) ?(?P<unit>.+)?'
 scaling_factor_regex_str = r'^(?P<axis>x|y)(_scaling_factor|sf)\: (?P<value>-?\d+\.?\d*)'
+curve_regex_str = r'^curve: ?(?P<curve_id>[\W|\w]*)'
```

Comment:
What is `\w|\W`? Is this not the same as `.`?

Reply:
I think you are right. Changed.